src/Mod/Path/PathScripts/PathSanity.py

```diff
@@ -464,7 +465,9 @@ def __report(self, data):
             )
 
         try:
-            result = os.system("asciidoctor {} -o {}".format(reportraw, reporthtml))
+            result = subprocess.Popen(
+                "asciidoctor {} -o {}".format(reportraw, reporthtml)
```

Comment:
Popen should take an array of arguments, not a pre-formatted string. The example from the manual is:
```
Popen(["/usr/bin/git", "commit", "-m", "Fixes a bug."])
```

Reply:
Well that explains a lot.  resolved.